Draft-2018-TTWG-Charter.html

```diff
@@ -589,7 +589,7 @@ <h2> Decision Policy </h2>
           face-to-face meeting or teleconference will be considered provisional.
           A call for consensus (CfC) will be issued for all resolutions (for
           example, via email and/or web-based survey), with a response period
-          from one week to 10 working days, depending on the chair's evaluation
+          of 10 working days, depending on the chair's evaluation
           of the group consensus on the issue. If no objections are raised on
           the mailing list by the end of the response period, the resolution

```

Comment:
Shouldn't the following be removed now?

_, depending on the chair's evaluation   of the group consensus on the issue._

Reply:
Yes, please remove the `, depending on the chair's evaluation of the group consensus on the issue` since it is no longer applicable.